Runtime 10.8 + 9.10.2 + 9.13 release notes (#3655)
astronomer/docs: astro/runtime-release-notes.md

- Fixed a bug where Airflow would show `failed_upstream` when a dynamically mapped task was `skipped`.
- In the Python task decorator, you can only have `None` as the default parameter for context parameters.

## Astro Runtime 10.8.0

- Release date: April, 18 2024
- Airflow version: 2.8.4

### Additional improvements

> - Updated `sqlparse` to `0.5.0`.

Author: ```suggestion
- Added functionality for using plugins to generate custom views in the Airflow UI. This feature will be fully available on Astro in a future release. 
- Updated `sqlparse` to `0.5.0`.
```

- Upgraded [Gunicorn](https://gunicorn.org/) to `22.0.0`.

> 

Reviewer: We need to include the changes for PR - https://github.com/astronomer/astro-runtime/pull/1426

- Fixed a bug where Airflow would show `failed_upstream` when a dynamically mapped task was `skipped`.
- In the Python task decorator, you can only have `None` as the default parameter for context parameters.

## Astro Runtime 10.8.0

- Release date: April, 18 2024
- Airflow version: 2.8.4

### Additional improvements

> - Added functionality for using plugins to generate custom views in the Airflow UI. This feature will be fully available on Astro in a future release. 

Author: ```suggestion
- Added functionality for using plugins to generate custom menu items in the Airflow UI. This feature will be fully available on Astro in a future release. 
```

- Fixed an issue in Astro where logging features could be disrupted if you set `AZURE_CLIENT_ID` as an environment variable.
- Fixed an issue where Astro audit logs listed a user's name as `User` for trigger events instead of their IDs.

## Astro Runtime 9.13.0

- Release date: April, 18 2024
- Airflow version: 2.7.3

### Additional improvements

- Upgraded [Gunicorn](https://gunicorn.org/) to `22.0.0`.
- Updated the version of `sqlparse` to `0.5.0`.

> 

Reviewer: We need to include the changes for PR - https://github.com/astronomer/astro-runtime/pull/1426

> 

Author: ```suggestion
- Added functionality for using plugins to generate custom views in the Airflow UI. This feature will be fully available on Astro in a future release. 

```

- Fixed an issue in Astro where logging features could be disrupted if you set `AZURE_CLIENT_ID` as an environment variable.
- Fixed an issue where Astro audit logs listed a user's name as `User` for trigger events instead of their IDs.

## Astro Runtime 9.13.0

- Release date: April, 18 2024
- Airflow version: 2.7.3

### Additional improvements

> - Added functionality for using plugins to generate custom views in the Airflow UI. This feature will be fully available on Astro in a future release. 

Author: ```suggestion
- Added functionality for using plugins to generate custom menu items in the Airflow UI. This feature will be fully available on Astro in a future release. 
```

> - Upgraded [Gunicorn](https://gunicorn.org/) to `22.0.0`.
> - Updated the version of `sqlparse` to `0.5.0`.
> - Added functionality for using plugins to generate custom views in the Airflow UI. This feature will be fully available on Astro in a future release. 

Author: ```suggestion
- Added functionality for using plugins to generate custom views in the Airflow UI. This feature will be fully available on Astro in a future release. 
- Upgraded [Gunicorn](https://gunicorn.org/) to `22.0.0`.
- Updated the version of `sqlparse` to `0.5.0`.
```
